src/vite-env.d.ts

```diff
@@ -46,3 +46,7 @@ declare module '@agoric/inter-protocol/src/interest-math' {
     currentCompoundedInterest: Ratio,
   ) => Amount<'nat'>;
 }
+
+declare module 'react-view-slider' {
+  export const ViewSlider;
```

Comment:
I think if this described the type being exported you wouldn't need the ts-expect-error cast when you import it

Reply:
Unfortunately it doesn't seem to be respecting it. I get `[JSX element type 'ViewSlider' does not have any construct or call signatures](https://github.com/s-yadav/react-number-format/issues/290#top)` unless I cast it here.